tests/dummy/app/templates/docs/queries.md

```diff
@@ -46,8 +46,8 @@ public methods:
 
 | Name           | Type              | Description                                                  |
 | -------------- | ----------------- | ------------------------------------------------------------ |
-| `filter`       | `String | Object` | According to the `gql` filter type defined in the query you're using. |
-| `order`        | `String | Object` | According to the `gql` order type defined in the query you're using. |
+| `filter`       | `String` | `Object` | According to the `gql` filter type defined in the query you're using. |
+| `order`        | `String` | `Object` | According to the `gql` order type defined in the query you're using. |
```

Comment:
This was correct. Why did you change it?

Reply:
Didn't seem correct to me, the words weren't formatted with the backticks

Reply:
Thats the correct notation in JSDoc for a type that can be a `String` *or* an `Object`

Reply:
![image](https://user-images.githubusercontent.com/30687616/119845286-6c72b600-bf09-11eb-803b-8571d6e63da0.png)
But it gets interpreted as a line of the table and cuts off the last column of text. My solution is also not correct, but thats the issue I found with that notation.

Reply:
Ah, just escape it then: \`String\|Object\`

Reply:
Changed